core/src/main/java/com/linecorp/armeria/client/endpoint/dns/DnsTextEndpointGroupBuilder.java

```diff
@@ -33,11 +33,10 @@
     /**
      * Creates a new instance that builds a {@link DnsTextEndpointGroup} for the specified {@code hostname}.
      *
-     * @param hostname the hostname to query DNS queries for
-     * @param mapping the {@link Function} that maps the content of a {@code TXT} record into
-     *                an {@link Endpoint}. The {@link Function} is expected to return {@code null}
-     *                if the record contains unsupported content.
```

Comment:
Could we copy these `@param` tags to the `builder()` method? Please fix this for all changes for this PR.

Reply:
Ok, I will revert all deletion of @param statements that I have deleted.